lib/metadataTypes/Automation.js

```diff
@@ -230,12 +230,11 @@ class Automation extends MetadataType {
      * @param {TYPE.AutomationMap} metadata metadata mapped by their keyField
      * @param {string} targetBU name/shorthand of target businessUnit for mapping
      * @param {string} retrieveDir directory where metadata after deploy should be saved
+     * @param {boolean} [isRefresh] optional flag - so far not used by automation
      * @returns {Promise.<TYPE.AutomationMap>} Promise
      */
-    static async deploy(metadata, targetBU, retrieveDir) {
```

Comment:
sure this doesnt break something?

Reply:
at least in my tests things kept working afterwards. 
i did a find-in-folder for postDeployTasks, upsert and deploy to find all occurrences across classes